epub33/a11y/index.html

```diff
@@ -915,83 +925,96 @@ <h6>Skippability</h6>
 										colours or placement so readers can filter this information visually out while
 										reading).</p>
 									<p>Readers who prefer auditory playback, however, cannot skip this information with
-										the same ease by default. Media overlays playback will typically result in
-										Reading Systems rendering secondary content where it occurs.</p>
-									<p>When EPUB Creators add structural semantics to Media Overlay Documents, however,
-										Reading Systems can create reading experiences that allow users to decide which
-										secondary content to skip by default during playback.</p>
+										the same ease by default. Synchronized text-audio playback will typically result
+										in Reading Systems rendering secondary content where it occurs.</p>
+									<p>When EPUB Creators add structural semantics, however, Reading Systems can create
+										reading experiences that allow users to decide which secondary content to skip
+										by default during playback.</p>
 								</dd>
 
-								<dt id="sec-mo-skippability-conf">Meeting this Objective</dt>
+								<dt id="sec-sync-skippability-conf">Meeting this Objective</dt>
 								<dd>
-									<p>EPUB Creators SHOULD identify all <a
-											href="https://www.w3.org/TR/epub/#sec-skippability">skippable structures</a>
-										[[EPUB-3]] in Media Overlay Documents.</p>
+									<p>EPUB Creators SHOULD identify all skippable structures if the synchronized
+										text-audio playback format does not handle escaping content by default.</p>
```

Comment:
Oops, ya, the text was similar so I did some copying... 😄 

It made me wonder if we really need two different objectives, or if adding semantics is one requirement with two useful outcomes. But I don't want to do too much rethinking at this stage.

Saying "by default" is probably not the best phrasing, but I was thinking of the discussion we had earlier about using the content document to provide skippability and escapability. If a technology were to do that, why would we need to bother with this objective? Making sure tables are marked up as tables, lists as lists, etc. is already required by WCAG 1.3.1 Info and Relationships. We'd be duplicating it.

I'm not sure how to say "if the host grammar is used to identify escapable structures so you don't need any additional semantics" in a succinct way. We could ignore the problem for now, though, and revisit the objectve whenever an alternative like that becomes a reality.

Reply:
> or if adding semantics is one requirement with two useful outcomes

To me, this is a slightly better way to present it but the current way is probably just as good for people who will read this document.

> We could ignore the problem for now, though, and revisit the objective whenever an alternative like that becomes a reality

Yeah let's ignore it for now :) I think as things are currently, implementations expect duplicate semantics in the sync text/audio format rather than just going straight to the content doc source. 

I am experimenting a bit with the latter in [a web-based book project](https://daisy.github.io/accessible-books-in-browsers/demos/action-for-heroes/005.html), will report back for EPUB vNextNext 🙃

Reply:
The next commit will strip these sentences down to: "EPUB Creators SHOULD identify all (skippable/escapable) structures."